frontend/document/picdocument.lua

```diff
@@ -50,9 +50,9 @@ end
 
 function PicDocument:register(registry)
     registry:addProvider("gif", "image/gif", self, 100)
-    registry:addProvider("jpg", "image/jpeg", self, 100)
-    registry:addProvider("jpeg", "image/jpeg", self, 100)
-    registry:addProvider("png", "image/png", self, 100)
+    registry:addProvider("jpg", "image/jpeg", self, 80)
```

Comment:
In my mind 100 means "fully intended to open" so saying MuPDF is only 90 would be saying (to the code/maintainers) that there's still something wrong with it. I was just a bit shy about it when I added all those new picture formats I suppose. Anyway, I'll merge it like this since that's more of a separate issue.